infrastructure/cloudformation_template.yaml

```diff
@@ -0,0 +1,163 @@
+AWSTemplateFormatVersion: '2010-09-09'
```

Comment:
Tested on CloudFormation, connection between resources, need to bump up instance type for EC2 and RDS to run.